lib/src/error.ts

```diff
@@ -80,3 +80,10 @@ export class ConfirmStepError extends ExchangeError {
     this.name = "ConfirmStepError";
   }
 }
+
+export class PayinExtraIdError extends ExchangeError {
+  constructor(nestedError?: Error) {
+    super("swap010", nestedError);
+    this.name = "PayinExtraIdError";
+  }
+}
```

Comment:
When does this error is trigger? when sending wrong tx to LL ?

Reply:
This one is trigged when creating a `stellarTransaction` or `rippleTransaction` and the `payinExtraId` is missing 
`if (!payinExtraId) throw new PayinExtraIdError();`